java/wavefrontsender.md

```diff
@@ -5,103 +5,158 @@
+* The HTTP URL of your Wavefront instance. This is the URL you connect to when you log in to Wavefront, typically something like `http://<domain>.wavefront.com`.
+* [Obtain the API token](http://docs.wavefront.com/wavefront_api.html#generating-an-api-token).
 
->**Prerequisite**
-> * Verify that you have the Direct Data Ingestion permission. For details, see [Examine Groups, Roles, and Permissions](https://docs.wavefront.com/users_account_managing.html#examine-groups-roles-and-permissions).
-> * The URL of your Wavefront instance. This is the URL you connect to when you log in to Wavefront, typically something like `https://<domain>.wavefront.com`.
-> * [Obtain the API token](http://docs.wavefront.com/wavefront_api.html#generating-an-api-token).
+### Initialize the WavefrontClient
 
-### Initialize the WavefrontDirectIngestionClient
-You initialize a `WavefrontDirectIngestionClient` by building it with the access information you obtained obtained in the Prerequisite section.
+You initialize a `WavefrontClient` by building it with the information you obtained in the Prerequisites section.
 
-You can optionally call builder methods to tune the following ingestion properties:
+Optionally, you can call factory methods to tune the following ingestion properties:
 
-* Max queue size - Internal buffer capacity of the `WavefrontSender`. Any data in excess of this size is dropped.
+* Max queue size - Internal buffer capacity of the `WavefrontSender`. Data that exceeds this size is dropped.
 * Flush interval - Interval for flushing data from the `WavefrontSender` directly to Wavefront.
 * Batch size - Amount of data to send to Wavefront in each flush interval.
 
-Together, the batch size and flush interval control the maximum theoretical throughput of the `WavefrontSender`. Override the defaults _only_ to set higher values.
+Together, the batch size and flush interval control the maximum theoretical throughput of the `WavefrontSender`. 
+> **Note**: Override the defaults values only when you want to set higher values.
+
+**Example**: Use a factory class to create a WavefrontClient and send  data to Wavefront via direct ingestion.
+
+```java
+// Create a factory and add a client with the following URL format: "http://TOKEN@DOMAIN.wavefront.com"
+// and a Wavefront API token with direct ingestion permission
+WavefrontClientFactory wavefrontClientFactory = new WavefrontClientFactory();
+
+// Add a new client that sends data directly to Wavefront services 
+wavefrontClientFactory.addClient(wavefrontURL,
+  20_000,           // This is the max batch of data sent per flush interval
+  100_000,          // This is the size of internal buffer beyond which data is dropped
+  2,                // Together with the batch size controls the max theoretical throughput of the sender
+  Integer.MAX_VALUE // The maximum message size in bytes we will push with on each flush interval 
+);
```

Comment:
The example code for proxy client factory doesn't include the optional properties.  I think we should be consistent with that one -- whether it's including the properties or not in both.  I'm more inclined to keep the example simple and exclude the properties, especially since they are called out a few lines above already.